docs/_sidebar.md

```diff
@@ -20,62 +18,46 @@
   * [Tokenomics](interlay/tokenomics.md)
 -->
 
-* **Kintsugi Canarynet <br/> (Kusama)**
-
+* **Kintsugi Canarynet (Kusama)**
 
   * [Overview](kintsugi/overview.md)
   * [Crowdloans](kintsugi/crowdloans.md)
   * [Tokenomics](kintsugi/tokenomics.md)
   * [Governance](kintsugi/governance.md)
 
-
 * **Guides (Testnet)**
 
-
   * [Setup](guides/prereq.md)
   * [Issue interBTC](guides/issue.md)
   * [Redeem interBTC](guides/redeem.md)
 
-
 * **Vaults**
 
-
   * [Overview](vault/overview.md)
   * [Installation](vault/installation.md)
   * [Upgrading](vault/upgrading.md)
   * [Operating Guide](vault/guide.md)
   * [FAQ](vault/faq.md)
 
-
 * **Collators**
 
-
   * [Overview](collator/overview.md)
   * [Installation](collator/guide.md)
   * [FAQ](collator/faq.md)
 
-
-
 * **Developers**
 
-
   * [interbtc.js](developers/integration.md)
   * [References](developers/documentation.md)
-  
-
 
 * **Further Reading**
 
-
 * [Roadmap](about/roadmap.md)
 * [Research](about/research.md)
 * [Glossary](about/glossary.md)
-* [Formal Specification :arrow_upper_right:](https://spec.interlay.io/)
-
+* [Formal Specification 🡥](https://spec.interlay.io/)
```

Comment:
```suggestion
* [Formal Specification  :notebook_with_decorative_cover:](https://spec.interlay.io/)
```

Reply:
I used the error to indicate an external link. Not sure what the emoji should do?